query/read/fields.go

```diff
@@ -37,6 +38,37 @@ func BuildFields(reqFields jsoniter.RawMessage) (*FieldFactory, error) {
 	factory.Include = make(map[string]Field)
 	factory.Exclude = make(map[string]Field)
 
+	var v interface{}
```

Comment:
allowing read fields input to be an array, will update user facing documentation for both APIs separately